themes/hugo-bootstrap-5/layouts/tcf/tcf-day-of.html

```diff
@@ -0,0 +1,500 @@
+        <p>
+          UBC CSSS
+          <br />
+          021 - 2366 Main Mall
+          <br />
+          Vancouver, BC
+          <br />
+          CA, V6T1Z4
+        </p>
+      </div>
+      <hr />
+
+      <div id="building-details">
+        <h2>Building Details</h2>
+        <p>
+          We are located in the Great Hall on the 2nd floor of the Nest, which
+          is located on the side of the Nest closest to Brown's Clubhouse and
+          the University Blvd cul-de-sac on the second floor.
+        </p>
+        <p>
+          We will attach a video closer to the date to help you find the
+          location.
+        </p>
+        <p>
+          We recommend using the elevator by the Great Hall for the fastest
+          movement of materials.
+        </p>
+        <p>
+          The map of the building and relevant ammenties for the first two
+          floors can be found here:
+        </p>
+        <img width="100%" src="/files/tcf2024/TCF-2024-Nest-Floor-1.png" />
+        <img width="100%" src="/files/tcf2024/TCF-2024-Nest-Floor-2.png" />
+      </div>
+      <hr />
+
+      <div id="venue-details">
+        <h2>Venue Details</h2>
+        <p>
+          The Great Hall is a large open space with a stage at the front and a
+          large open area for booths. <br />
+          There are no walls or dividers between booths, so please be respectful
+          of your neighbours.<br />
+          We are working with building staff to supply power to all booths via
+          extension cords. We are to be given at least 2 outlets per employer.
+          <br />
+          We have been told that the estimated amperage is 20. <br />
+          Wifi should be accessible through UBC Public Wifi. See the Internet
+          Access section below for instructions. <br />
+          There are no coat racks so please keep this in mind when packing.
+          <br />
+          We will provide a locked and empty room to store any materials. Please
+          contact a volunteer on the day for access as it requires a fob.
+        </p>
+        <p>
+          Below is the following floor map for companies. This is subject to
+          change as we work with the catering staff. <br />
+          Company locations will be marked with a paper on the tables with the
+          company name on the day.
+        </p>
+
+        <h2 id="venue-map" style="text-align: center;">Venue Map</h2>
+        <img width="100%" src="/files/tcf2024/TCF-2024-Venue-Map.png" />
+        <hr />
+
+        <h2 id="booth-assignments" style="text-align: center;">
+          Booth Assignments
+        </h2>
+        <p style="text-align: center;">
+          The following booth assignments are subject to change. Check back on
+          the day for the final assignments
+        </p>
+        <div class="grid-container">
+          <div class="row pt-4">
+            {{ $.Scratch.Set "counter" 0 }}
```

Comment:
this is a temporary fix. I will add to the company markdowns the true venue numbers next week. Rn is not the time